src/components/SearchResultContainer/SearchResultContainer.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Row, Col, Button, Empty } from 'antd';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faChevronRight,
+  faMapMarkerAlt,
+  faTree,
+} from '@fortawesome/free-solid-svg-icons';
+import { groupBy } from 'lodash';
+import { LearnAndEarnIcons, TitleDivider } from 'components/shared';
+
+export default function ({ data = [], toggeables, setToggeables }) {
+  const group = groupBy(data, 'entity_type');
+  const groupKeys = Object.keys(group);
+
+  return (
+    <div className="h-full">
+      {data && data.length && <p className="text-2xl text-center">Results</p>}

```

Comment:
move this up to a early return

if (!data.length) return some kind of empty data logo